site/en/style-guide/index.njk

```diff
@@ -1,5 +1,10 @@
 ---
 title: Style guide
+banner:
+  text: 'This is a banner'
+  action: '/style-guide'
```

Comment:
These feel around the wrong way. `action_href` reads to me like it should be the target URL, not the text.